posthog/models/user.py

```diff
@@ -78,6 +78,9 @@ class User(AbstractUser):
     anonymize_data: models.BooleanField = models.BooleanField(
         default=False, null=True, blank=True
     )
+    toolbar_mode: models.CharField = models.CharField(
```

Comment:
Are you thinking we'll have different modes here? Might be worth adding those specifically [as choices](https://docs.djangoproject.com/en/3.0/ref/models/fields/#choices)

Reply:
I was thinking to reuse this field in the future and support values like "float" and "dock", to specify which would be the default mode for the "big floating button" that opens the toolbar.

So for now the 2 enums that this field uses are "default" and "toolbar". In the future I'd expand it to include "toolbar.float", "toolbar.dock" and who knows what else we can come up with.

Not sure if that makes sense or not. I can replace it with a bool for now and refactor/migrate it to a choice once we have a choice to give. The float/dock separation could even be in a new field then...

Reply:
Okay worth adding those two enums as Choices in that case, just so we don't end up with random stuff in there accidentally

Reply:
Choices added! They will for sure change when we release the toolbar, so I wouldn't put too much emphasis on having them perfect now.